tests/unit/api/bucketDeleteLifecycle.js

```diff
@@ -50,7 +50,7 @@ describe('deleteBucketLifecycle API', () => {
             next => metadata.getBucket(bucketName, log, next),
         ], (err, results) => {
             assert.equal(err, null, `Expected success, got error: ${err}`);
-            const bucket = results[2];
+            const bucket = results[2][0];
```

Comment:
seems weird that only this line is updated : shouldn't there be a change as well in cloudserver code, if API changed in arsenal?